pages/terra/autoupdate.mdx

```diff
@@ -39,7 +39,8 @@ Updates are triggered every 10 minutes using `anda update`.
 #### Nightly packages
 
 Nightly packages are updated every 24 hours using
-`anda update -vv --filters nightly=1`. Their `anda.hcl` files should look something like this:
+`[1;32manda[97m update [0;3;37m-vv --filters[0;97;m nightly=1{:ansi}`.
```

Comment:
what does this render to?

Reply:
https://3e88767c.devdocs-next.pages.dev/terra/autoupdate#nightly-packages